sys-process/bottom/bottom-0.6.3.ebuild

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.3.2
+
+EAPI=7
```

Comment:
you can make it `EAPI=8`
it will force rust-1.53 dependency though, which is not stable yet. but it will go stable soon.